tests/unit/test_openslide.py

```diff
@@ -1,12 +1,18 @@
+import random
+
 import tiledb
 
-from tests import get_CMU_1_SMALL_REGION_schemas
-from tiledbimg.openslide import LevelInfo, TileDBOpenSlide
+from tests import get_schema
+from tiledbimg.openslide import TileDBOpenSlide
 
 
 class TestTileDBOpenSlide:
     def test_from_group_uri(self, tmp_path):
-        schemas = get_CMU_1_SMALL_REGION_schemas()
+        def r():
+            return random.randint(64, 4096)
+
+        level_dimensions = [(r(), r()) for _ in range(7)]
```

Comment:
Can you add a comment here on why hard coding `range(7)`

Reply:
No particular reason, I can make this random too

Reply:
[Done](https://github.com/TileDB-Inc/TileDB-BioImaging/pull/29/commits/3832f82dc7ce59d0b1a4dcddaafd50edde45aaf1)